mantidproject/mantid
# In LoadIsawDetCal, use the standard library instead of handwritten loops .

Description of work.

I found two places in `LoadIsawDetCal.cpp` where standard library functions can be used instead of hand-written loops. Rather than looping through the entire container, both of these functions should return early.

**To test:**



Verify that these changes don't effect the results.

This is a small change with no issue number.

Does not need to be in the release notes.

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
